README.md

```diff
@@ -36,96 +36,24 @@ pip3 install .
-  ```
-  result = vim.api.strwidth("some text")
-  ```
-  Note the initial `nvim_` is not included. Also, object methods can be called
-  directly on their object,
-  ```
-  buf = vim.current.buffer
-  len = buf.api.line_count()
-  ```
-  calls `nvim_buf_line_count`. Alternatively msgpack requests can be invoked
-  directly,
-  ```
-  result = vim.request("nvim_strwith", "some text")
-  len = vim.request("nvim_buf_line_count", buf)
-  ```
-
-* The API is not thread-safe in general. However, `vim.async_call` allows a
-  spawned thread to schedule code to be executed on the main thread. This method
-  could also be called from `:python` or a synchronous request handler, to defer
-  some execution that shouldn't block nvim.
-  ```
-  :python vim.async_call(myfunc, args...)
-
-  ```
-  Note that this code will still block the plugin host if it does long-running
-  computations. Intensive computations should be done in a separate thread (or
-  process), and `vim.async_call` can be used to send results back to nvim.
-
-* Some methods accept an `async_` keyword argument: `vim.eval`, `vim.command`,
-  `vim.request` as well as the `vim.funcs` and `vim.api` wrappers.  When
-  `async_=True` is passed the client will not wait for nvim to complete the
-  request (which also means that the return value is unavailable).
-
-#### Remote (new-style) plugins
-
-Neovim allows python3 plugins to be defined by placing python files or packages
-in `rplugin/python3/` (in a runtimepath folder). Python2 rplugins are also
-supported and placed in `rplugin/python/`, but are considered deprecated.
-Further added library features will only be available on python3. Rplugins follow
-the structure of this example:
+Pynvim supports both python plugins using the language-agnostic rplugin
+interface, as well as vim plugins
+using the `:python[3]` interface. Thus, when pynvim is installed, neovim will
+report support for the `+python[3]` vim feature.
 
-```python
-import neovim
-
-@neovim.plugin
-class TestPlugin(object):
-
-    def __init__(self, nvim):
-        self.nvim = nvim
-
-    @neovim.function("TestFunction", sync=True)
-    def testfunction(self, args):
-        return 3
-
-    @neovim.command("TestCommand", range='', nargs='*')
-    def testcommand(self, args, range):
-        self.nvim.current.line = ('Command with args: {}, range: {}'
-                                  .format(args, range))
-
-    @neovim.autocmd('BufEnter', pattern='*.py', eval='expand("<afile>")', sync=True)
-    def on_bufenter(self, filename):
-        self.nvim.out_write("testplugin is in " + filename + "\n")
-```
+The rplugin inteface allows plugins to handle vimL function calls as well as
+defining commands and autocommands, and such plugins can operate asynchronously
+without blocking nvim.  For details of using the new rplugin interface, please
+see the [Remote plugin](http://pynvim.readthedocs.io/en/latest/usage/remote-plugins.html) documention.
```

Comment:
brevity: "For details ~~of using~~ on the new rplugin interface, ~~please~~ see the"